File: src/main/java/de/tectoast/emolga/commands/music/control/LoopCommand.java
package de.tectoast.emolga.commands.music.control;

import de.tectoast.emolga.commands.GuildCommandEvent;
import de.tectoast.emolga.commands.MusicCommand;
import de.tectoast.emolga.utils.music.GuildMusicManager;

public class LoopCommand extends MusicCommand {

    public LoopCommand() {
        super("loop", "Loopt den derzeitigen Track oder beendet die Loop");
        setArgumentTemplate(ArgumentManagerTemplate.noArgs());
    }

    @Override
    public void process(GuildCommandEvent e) {
        GuildMusicManager manager = getGuildAudioPlayer(e.getGuild());
        e.reply("Loop wurde " + (manager.scheduler.toggleLoop() ? "" : "de") + "aktiviert!");
    }
}

File: src/main/kotlin/de/tectoast/emolga/commands/music/control/LoopCommand.kt
package de.tectoast.emolga.commands.music.control

import de.tectoast.emolga.commands.GuildCommandEvent
import de.tectoast.emolga.commands.MusicCommand

class LoopCommand : MusicCommand("loop", "Loopt den derzeitigen Track oder beendet die Loop") {
    init {
        argumentTemplate = ArgumentManagerTemplate.noArgs()
    }

    override fun process(e: GuildCommandEvent) {
        e.reply("Loop wurde ${if (getGuildAudioPlayer(e.guild).scheduler.toggleLoop()) "" else "de"}aktiviert!")
    }
}